{{cookiecutter.project_dirname}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -307,6 +307,12 @@ class Testing(ProjectDefault):
 
     # MEDIA_ROOT = ProjectDefault.BASE_DIR / "media_test"{% endif %}
 
+    # unittest-xml-reporting (aka xmlrunner)
+    # https://github.com/xmlrunner/unittest-xml-reporting#django-support
+
+    TEST_OUTPUT_FILE_NAME = "report.xml"
+    TEST_RUNNER = "xmlrunner.extra.djangotestrunner.XMLTestRunner"
```

Comment:
A space is missing between lines 313 and 314.

Reply:
blank line added